index.html

```diff
@@ -85,242 +95,270 @@ <h2>LET'S CONNECT</h2>
+            <li>
+              <a href="#footer-icon">
+                <img src="./images/Linkedin icon.svg" alt="Linkedin logo"
+              />
+              </a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/hand-peace-icon.svg" alt="peace logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/twitter (2).svg" alt="twitter logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/m-icon.svg" alt="m logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <button  id="btn-get" class="see-pro .btn">Get my resume</button>
       </div>
-    </section>
-</fieldset>
+       <div class="desktopV">
+
+         <div class="frame-lang">
+           <h2 class="language">Languages</h2>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+         <div class="list-skills-about">
+           <ul>
+             <li>
+              <img class="img-tanne" src="./images/javascript.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/JSdesktopAbout.svg" alt="" />
+             </li>
+             <li>
+              <img class="img-tanne" src="./images/html.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/HTMLdesktopAbout.svg" alt="" />
+            </li>
+             <li>
+              <img class="img-tanne" src="./images/css.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/CSSdesktopAbout.svg" alt="" />
+           </ul>
+         </div>
+         <div class="frame">
+           <p>Frameworks</p>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+         <div class="frame">
+           <p class="skill-desktop">Skills</p>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+       </section>
+
+       </div>
+    <!-- ============CONTACT-ME-FORM==========  -->
+    <fieldset>
+      <section class="contact-me">
+       <div class="text-field hello">
+         <h2>Contact me</h2>
+         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+       </div>
+       <div class="input-field">
+         <form action="https://formspree.io/f/mnqynlww" method="post">
+           <input class="input-design" type="text" name="full_name" id="full_name" placeholder="Enter your Name" maxlength="30" required>
+           <input class="input-design" type="email" name="email" id="email" placeholder="Email Address" required>
+           <textarea class="textArea" name="message"  cols="30" rows="10" minlength="" maxlength="500" placeholder="write your message here" required></textarea>
+           <input  type="submit" value="Get in touch" class="see-pro submit-btn">
```

Comment:
- [ ] The inputs setup is very interesting and all the HTML requirements are met ☺️ . However, for the CSS part, ensure to stick to the Figma design as much as possible. I suggest you consider reducing the `border-radius` of the inputs to make it as same as that of the Figma design 😎 

  - Your design 😎 
![input](https://user-images.githubusercontent.com/99394374/215087065-14296f76-5e7a-4cd2-8b5e-bb73fa08f953.PNG)

  - Figma design 😎 
![input fig](https://user-images.githubusercontent.com/99394374/215087141-dc4afb72-7cfd-42b9-8bd2-f4fcbdf07aa2.PNG)